jenkins/ci.suse.de/deploy-crowbar-poc

```diff
@@ -29,6 +29,8 @@ export nodenumber=2
 
 case "$1" in
   pre)
+    # workaround: remove everything but eth0
+    rm /etc/sysconfig/network/ifcfg-eth[1-9]*
```

Comment:
Shouldn't we drop this commit, because these file get deleted via the Jenkinsfile?

Reply:
done